File: python/7_kyu/vowel_count.py
# Vowel Count
# https://www.codewars.com/kata/54ff3102c1bad923760001f3/solutions/python

vowels = ['a', 'e', 'i', 'o', 'u']

def get_count(input_str):
    count = 0
    
    for i in input_str:
        if i in vowels:
            count += 1
    
    return count

File: javascript/7_kyu/vowel_count.js
// Vowel Count
// https://www.codewars.com/kata/54ff3102c1bad923760001f3/train/javascript

const vowels = ['a', 'e', 'i', 'o', 'u'];

function getCount(str) {
  var count = 0;
  
  for (var i = 0; i < str.length; i++) {
    if ( vowels.includes(str[i]) ) {
      count++;
    }
  }
  
  return count;
}